lib/ae_mdw/sync/async_tasks/consumer.ex

```diff
@@ -1,17 +1,40 @@
 defmodule AeMdw.Sync.AsyncTasks.Consumer do
   @moduledoc """
-  Synchronization tasks that run asynchronously consuming Model.AsyncTasks records.
+  Database Sync tasks that run asynchronously consuming Model.AsyncTasks records.
   """
   use GenServer
 
-  alias AeMdw.Db.Contract
   alias AeMdw.Db.Model
-  alias AeMdw.Node.Db, as: DBN
+  alias AeMdw.Log
+
   alias AeMdw.Sync.AsyncTasks.Producer
+  alias AeMdw.Sync.AsyncTasks.TaskSupervisor
+  alias AeMdw.Sync.AsyncTasks.UpdateAex9Presence
 
   require Model
+  require Logger
+
+  import AeMdw.Util, only: [ok!: 1]
+
+  @base_sleep_msecs 3_000
+  @yield_timeout_msecs 100
+  @task_timeout_msecs 30_000
```

Comment:
changed from 20s to 30s as the counter of long tasks reached 5 (different contracts).